src/macaron/slsa_analyzer/checks/provenance_l3_check.py

```diff
@@ -202,10 +230,11 @@ def run_check(self, ctx: AnalyzeContext, check_result: CheckResult) -> CheckResu
         """
         # TODO: During verification, we need to fetch the workflow and verify that it's not
         # using self-hosted runners, custom containers or services, etc.
-        all_feedback = []
+        all_feedback: list[tuple[str, str, _VerifyArtefactResult]] = []
```

Comment:
How about using [`typing.NamedTuple`](https://docs.python.org/3/library/typing.html#typing.NamedTuple) instead of a plain `tuple` to improve the readability and prevent potential mistakes in looking up fields?

Reply:
That's perfect thanks I didn't know that existed

Reply:
This is addressed in [78b65b2](https://github.com/oracle-samples/macaron/pull/29/commits/78b65b23f3b61676ad407205fc9d7bee39fa7b99)

Reply:
Thanks, LGTM. Let's close this PR and move the changes to https://github.com/oracle-samples/macaron/pull/46 as discussed.